### ISSUE TITLE
Bad title and Tab order messed up in New Identity dialog

### ISSUE THREAD

**Issue by dnotestein** [InvictusInnovations/keyhotee#170]
Change title to "Create New Identity".
Tab order should be left to right, then downwards.

**Comment by dnotestein** [comment]
Not reporting status properly. For example, doesn't report if ID was already registered by someone else.
